otherlibs/win32unix/link.c

```diff
@@ -36,7 +36,7 @@ CAMLprim value unix_link(value follow, value path1, value path2)
   tCreateHardLink pCreateHardLink;
   BOOL result;
   wchar_t * wpath1, * wpath2;
-  if (Is_block(follow) && !Bool_val(Field(follow, 0))) { /* Some false */
+  if (Is_some(follow) && ! Bool_val(Some_val(follow))) {
```

Comment:
```suggestion
  if (Is_some(follow) && !Bool_val(Some_val(follow))) {
```
(there's a mix of styles, but no space after `!` is the more common - FWIW I'd change L59 if at all)